tegel/src/components/theme/colour/colour-semantic.stories.js

```diff
@@ -0,0 +1,34 @@
+export default {
+  title: 'Foundations/Colour',
+  parameters: {
+    layout: 'fullscreen',
+  },
+};
+
+const Template = () => `
+  <style>
+  .demo-wrapper {
+    height: 90px;
+  }
+  .demo-wrapper span {
+    color: white;
+    background-color: black;
+    border: 1px solid white;
+    padding: 4px;
+    position: absolute;
+  }
+</style>
+  <div class="demo-wrapper" style="background-color: var(--sdds-positive)">
+    <span>positive</span>
```

Comment:
```suggestion
    <span>--sdds-positive</span>
```
same here as above ☝️